core-bundle/src/InsertTag/InsertTagParser.php

```diff
@@ -447,16 +448,27 @@ private function renderBlockSubscription(InsertTag $tag, ParsedSequence|null $co
 
     private function getFragmentForTag(InsertTag $tag): InsertTagResult
     {
+        $clientCache = 0;
+        $pageId = null;
+
+        $request = $this->requestStack->getCurrentRequest();
```

Comment:
Should this also use the `PageFinder`?

Reply:
The service is not internal, so we cannot add a dependency. It also does not make sense to inject both the request stack and the page finder, because both provide the page model.